system/Database/MigrationRunner.php

```diff
@@ -161,7 +161,7 @@ public function latest(?string $group = null)
 
         $this->ensureTable();
 
-        if ($group !== null) {
+        if ((string) $group !== '') {
```

Comment:
What is wrong with `null`?

Reply:
Using default group instance is reserved for `null` value, not an empty string. 

Strict typing becomes loose typing, why? Shouldn't we go in the opposite direction?